tests/set-labels/both-additionalLabelFields-fieldSpecs/Kptfile

```diff
@@ -6,3 +6,6 @@ pipeline:
   mutators:
     - image: gcr.io/kpt-fn/set-labels:unstable
       configPath: fn-config.yaml
+      selectors:
```

Comment:
Why do we use selectors here?

Reply:
The original file does not have selectors because it fails before applying the selectors. Here it is added because I modified the test to perform the actual operation (this test is duplicate coverage and should be merged with additional- labels later)